app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
+        raise ValueError("You line should start with cp")
+    if len(command) != 3:
```

Comment:
The error message 'Message should be: cp <source_file> <destination_file>' is not very clear. It would be better to say 'The command should be in the format: cp <source_file> <destination_file>'.